styles/blog.css

```diff
@@ -0,0 +1,188 @@
+* {
+   box-sizing: border-box;
+   font-family: Arial, Helvetica, sans-serif
+}
+
+/* header setting */
+header {
+  background-image: url(../images/header-image.jpg);
+  padding-left: 10%;
+  padding-top: 5%;
+  padding-right: 10%;
+  padding-bottom: 15%;
+}
+
+h1 {
+  color: #81B741;
+  font-family: "Lucida Handwriting", "Brush Script M7", cursive;
+  font-size: 150%;
+}
+
+.shopping-cart{
+  width: 1.5rem;
+  text-align: right;
+}
+
+.check-out {
+  background-color: #81B741;
+  color: white;
+  padding: 10px 4px;
+  text-align: center;
+}
+
+.headernavigation {
+    float: left;
+    display: block;
+    text-align: left;
+    align-content: center;
+    list-style-type: none;
+    overflow: hidden;
+    color: #083B60;
+    padding-left: 15%;
```

Comment:
for most padding and margin I like rem's (or px) things can look really odd at small widths